mozilla-release/browser/branding/cliqz/pref/firefox-branding.js

```diff
@@ -45,3 +45,4 @@ pref("browser.aboutHomeSnippets.updateUrl", "https://www.cliqz.com/");
 // Cliqz distribution settings
 pref("app.distributor", "cliqz");
 pref("mozilla.partner.id", "cliqz");
+pref("extensions.cliqz.integrated", "cliqz@cliqz.com,firefox@ghostery.com,https-everywhere@eff.org");
```

Comment:
this list can grow, can't we read them somehow? And ghostery is not always in, the user needs to install it manually from the prefs page

Reply:
plus can you please use a different namespace, other than extension.cliqz? maybe "browser.migration.soemthing"

Reply:
@luciancor, This list is not part of migration but allowing install from AMO, so IMO the namespace suits.
We are planning to get rid of installs from prefs, so probably we will not have Ghostery in the list. As I understand this list is of extensions pre-installed in browser and supposedly should not grow. We have separate feature `blocklist` which will take care of addons which will not be allowed to install.